integration_test/publish-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Could you please add a comment line for the parameters to the scripts on the second line similar to run-adapter-and-wait? It will improve the readability of the scripts (publish-images and stop-image)

Reply:
done

Reply:
Thanks, also I think publish var is missing from publish-image script.

Reply:
oh sorry
fixed